package.json

```diff
@@ -227,6 +216,19 @@
     "yakbak": "^3.0.0",
     "yargs": "^16.2.0"
   },
+  "devDependencies": {
+    "@commitlint/cli": "^12.0.0",
+    "@commitlint/config-conventional": "^12.0.0",
+    "documentation": "^13.1.1",
```

Comment:
Please update (if need be) the patch for `documentation`

Reply:
`documentation` version didn't change, it just became a dev dependency, so I don't think changes to the patch are necessary

Reply:
It did, it's currently at version `13.0.2` on master
https://github.com/webex/webex-js-sdk/blob/master/package.json#L131

It can be bumped, just the patch for it to use legacy decorators needs to be updated

Reply:
Oh, I didn't even notice the version change, thanks for catching it. I updated the patch